docsrc/index.rst

```diff
@@ -1,21 +1,7 @@
 ******************************
-indicoio Python Client Library
+Indico Python Client Library
```

Comment:
Should we add any intro or guidance on using the guide here? Probably worth asking technical writing?

Reply:
what sort of stuff do you think we should add? like information on how to install the sdk?